docs/swagger/spec.yml

```diff
@@ -5658,6 +5659,12 @@ parameters:
     in: query
     type: integer
     required: false
+  format:
+    name: format
+    description: "List tasks union by status if `summary` is specified"
```

Comment:
We're kind of inconsistent in how we format description lines, but looks like we are either using single quotes or no quotes -- might be good to change for consistency.